playbook-website/app/javascript/components/Website/src/pages/ComponentList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { NavLink, Outlet, useOutlet, useOutletContext } from "react-router-dom"
+import { Title } from "playbook-ui"
+
+export default function ComponentList() {
+  const outlet = useOutlet();
+  const isMobile = useOutletContext();
+
+  return (
+    <div className='pl_md'>
+      {!outlet && (
+        <NavLink to="avatar">
+          <Title paddingTop={isMobile ? "md" : ""} text={"Avatar"} size='2' />
+        </NavLink>
+      )}
+      <Outlet context={isMobile} /> 
+    </div>
```

Comment:
Can we clean this up in the next PR? I'd like to use a kit and a prop instead of direct classes

Reply:
Yes. I will do that in a follow-up story. 
This whole section will most likely be recoded, regardless. 
This is not final code, by any means.

Reply:
Fixed this @jasoncypret ! Used `<Flex>` kit instead!